js/main.js

```diff
@@ -97,11 +112,12 @@ document.addEventListener('DOMContentLoaded', function (event) {
   for (let i = 0; i < data.entries.length; i++) {
     $list.appendChild(renderEntry(data.entries[i]));
   }
-  viewSwap(data.view);
+  viewSwap('entries');
```

Comment:
This should be `data.view`. You don't want to swap to a specific page, you want to utilize whatever was stored on your last refresh